composer.json

```diff
@@ -24,6 +24,7 @@
         "php": "^5.6 || ^7.0.0",
         "symfony/event-dispatcher": "^2.3 || ^3.0 || ^4.0",
         "symfony/options-resolver": "^2.7 || ^3.0 || ^4.0",
+        "symfony/http-foundation": "^2.4 || ^3.0 || ^4.0",
```

Comment:
i think this should be in require-dev & suggest, just like http-kernel itself. its only needed when using the HttpCache (and i guess only when using the new client)

Reply:
I think it's okay to leave it here. 2.7 and 2.8 are the only supported Symfony version left of the 2.x branch. The 2.3 branch has lost its support for a year now. Nobody will be using http foundation lower than 2.4, I'm fairly certain of that :)

Reply:
i am not concerned about the version constraint, but about adding a new dependency that will be installed even when you don't use symfony at all.

Reply:
Okay will move to dev deps.